include/reach/reach_study.h

```diff
@@ -41,6 +41,7 @@ class ReachStudy
     float step_improvement_threshold;
     float radius;
     std::size_t max_threads = std::thread::hardware_concurrency();
+    std::map<std::string, double> initial_seed;
```

Comment:
Semantics, but let's change this to `seed_state`. The IK interface only provides a single seed state, and to me the name `initial_seed` indicates that other subsequent seed states might be provided, which is not currently the case
```suggestion
    std::map<std::string, double> seed_state;
```